### PR TITLE
*: changed to learning path style

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added, or deleted? (Required)

Changed the home page to learning path style

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/_index.md [hunk 1]
```diff
@@ -2,85 +2,80 @@
 title: TiDB in Kubernetes Documentation
 summary: Learn about TiDB in Kubernetes documentation.
 aliases: ['/docs/tidb-in-kubernetes/dev/']
+hide_sidebar: true
+hide_commit: true
 ---
 
-# TiDB in Kubernetes Documentation
+<LearningPathContainer platform="tidb-operator" title="TiDB in Kubernetes Documentation" subTitle="Using TiDB Operator provided by PingCAP, you can run and maintain TiDB seamlessly in the Kubernetes clusters deployed on a public cloud or on premise.">
```
Comment (ran-huang) [inline en/_index.md:9]: "[on-premises](https://andrewschoen.medium.com/software-people-of-the-world-stop-saying-on-premise-you-sound-dumb-56692dae871d#:~:text=%E2%80%9CON%2DPREMISE%E2%80%9D-,TLDR%3A%20It's%20%E2%80%9Con%2Dpremises%E2%80%9D%20or%20%E2%80%9Con%2D,say%20%E2%80%9Con%2Dpremise.%E2%80%9D)" is more accurate.
```suggestion
<LearningPathContainer platform="tidb-operator" title="TiDB in Kubernetes Documentation" subTitle="Using TiDB Operator provided by PingCAP, you can run and maintain TiDB seamlessly in the Kubernetes clusters deployed on a public cloud or on premises.">
```

File: en/_index.md [hunk 2]
```diff
@@ -2,85 +2,82 @@
...
-<ColumnTitle>About TiDB Operator</ColumnTitle>
+[What's New in v1.3](https://docs.pingcap.com/tidb-in-kubernetes/dev/whats-new-in-v1.3)
 
-- [TiDB Operator Overview](tidb-operator-overview.md)
-- [TiDB Operator Architecture](architecture.md)
-- [What's New in v1.3](whats-new-in-v1.3.md)
+[Get Started](https://docs.pingcap.com/tidb-in-kubernetes/dev/get-started)
 
-</NavColumn>
+[Relationship between TiDB Operator and TiDB Versions](https://docs.pingcap.com/tidb-in-kubernetes/dev/tidb-operator-overview)
```
Comment (ran-huang) [inline en/_index.md:21]: 这个关系还要加吗？

File: zh/_index.md [hunk 3]
```diff
@@ -2,88 +2,82 @@
...
+
+</LearningPath>
+
+<LearningPath label="参考" icon="cloud-dev">
+
+[API 参考文档](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.md)
+
+[工具](https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/tidb-toolkit)
+
+[Command Cheat Sheet](https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/cheat-sheet)
```
Comment (ran-huang) [inline zh/_index.md:79]:
```suggestion
[常用命令汇总](https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/cheat-sheet)
```

**Review by ran-huang — APPROVED** [review]
rest LGTM

File: en/_index.md (hunk 2 above)
Comment (hfxsd) [reply en/_index.md:21]: 按 Grace 的建议，放这里吧。

**Pull request merged**